build_envoy.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+mkdir -p build
+
+ENVOY_DEV_IMAGE=${1}
+
+CMD="-c 'CC=clang CXX=clang++ bazel --output_user_root=/build/ build //:envoy'"
+
+OPTIND=2
+while getopts "i" opt; do
+    case $opt in
+        i) CMD="" ;;
+    esac
+done
+
+docker run \
+       --rm \
+       -it \
```

Comment:
Is `-i` needed because this is non-interactive?

Reply:
@ipuustin 

Removing "-i" (detaching from STDIN) makes it cumbersome to interrupt it.

So, pehaps, let's keep it as a convenience?

Reply:
Fine with me.